docs/features/UsedAssemblyReferences.md

```diff
@@ -5,4 +5,8 @@ The *Used Assembly References* feature provides a ```GetUsedAssemblyReferences``
 of metadata assembly references that are considered to be used by the compilation. For example, if a type declared in a
 referenced assembly is referenced in source code within the compilation, the reference is considered to be used. Etc.
 
+Note that documentation must be processed for best results.
```

Comment:
Consider removing this statement or rephrasing in order to not use the "best result" term.

Reply:
Also, there is no requirement for "documentation must be processed". It is perfectly fine to use API without that.

Reply:
I prefer to keep that. You do get better results (more granular) when processing documentation.